src/playbacks/html5_video/html5_video.js

```diff
@@ -250,18 +252,6 @@ export default class HTML5Video extends Playback {
     this.el.volume = value / 100
   }
 
-  mute() {
```

Comment:
I'd rather keep this for now, since there could be some plugin implementations that rely on this. 

Perhaps instead of removing these methods, we could fix them to better reflect the `muted`/`defaultMuted` properties of the element.

Reply:
yes, that's what I thought. @tjenkinson they could call just the plugins `volume` method but then we d have to store a state of muted volume to recover it on unmute. I'd rather avoid that at this point.

maybe just have them set `this.el.muted` accordingly.

and also we could set them `@deprecated` and `@private` as docs decorator to show that this is not part of `Playback` API and that they may be removed in a minor version bump.

Reply:
I agree, we could just have those methods set `this.el.muted`, and make those methods `@deprecated` and `@private`.

We're still deciding whether or not to change the `Playback` API to include the `muted` state, but with a more concise interface.